packages/peregrine/lib/talons/AccountInformationPage/useAccountInformationPage.js

```diff
@@ -0,0 +1,122 @@
+import { useCallback, useMemo, useState, useEffect } from 'react';
+import { useMutation, useQuery } from '@apollo/client';
+import { useUserContext } from '../../context/user';
+
+export const useAccountInformationPage = props => {
+    const {
+        mutations: {
+            setCustomerInformationMutation,
+            changeCustomerPasswordMutation
+        },
+        queries: { getCustomerInformationQuery }
+    } = props;
+
+    const [{ isSignedIn }] = useUserContext();
+
+    const [activeChangePassword, setActiveChangePassword] = useState(false);
```

Comment:
Use `isFoo` syntax to denote a boolean value.

```suggestion
    const [isChangingPassword, setIsChangingPassword] = useState(false);
```